clicommand/secret_get.go

```diff
@@ -99,7 +99,7 @@ Examples:
 		}
 
 		agentClient := api.NewClient(l, loadAPIClientConfig(cfg, "AgentAccessToken"))
-		secrets, errs := secrets.FetchSecrets(ctx, agentClient, cfg.Job, cfg.Keys, 20)
+		secrets, errs := secrets.FetchSecrets(ctx, agentClient, cfg.Job, cfg.Keys, 10)
```

Comment:
10 should be fine.

Here's reddit's concurrent jobs over the past 7 days. Not exactly massive load.

<img width="1168" height="338" alt="image" src="https://github.com/user-attachments/assets/0bb981e6-1d32-46d1-8204-097b766de4ff" />